.asv/results/czipa1osx041/machine.json

```diff
@@ -0,0 +1,9 @@
+{
+    "arch": "x86_64",
+    "cpu": "Intel(R) Core(TM) i7-7567U CPU @ 3.50GHz",
+    "machine": "czipa1osx041",
+    "num_cpu": "4",
+    "os": "Darwin 18.5.0",
+    "ram": "17179869184",
+    "version": 1
```

Comment:
Yup, I was unsure about how to do this. Do we want a new repo with only the benchmark results? or with the benchmarking code too? I like having the benchmarking code in this repo, but am happy to make a new repo with just the results. I will do this unless I hear otherwise

There is also a whole lot of info on how to publish the results to a website etc. but I'm not sure we are there yet!!